docs/release_notes.rst

```diff
@@ -1,6 +1,21 @@
 Release Notes
 =============
 
+v3.2.0
+------
+
+API Additions
+^^^^^^^^^^^^^
+* Added ``content_type`` property for OBS paths representing consistent
+  Content-Type header that would be set on download.
+* Allow empty DXPath initialization. ``dx://`` is now a valid DXPath. ``stor ls dx://`` lists the
```

Comment:
since PR for this and the other one are still open, perhaps better to just update there?